### PR TITLE
Rebuild MicroStack home page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

Rebuild MicroStack home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit) and and resolve comments
- Check against [design](https://app.zeplin.io/project/611e22af1491325814021275/screen/611e22e82bc2085acc4bc1df)

## Issue / Card

Fixes [#4351](https://github.com/canonical-web-and-design/web-squad/issues/4351)

## Screenshots
MicroStack - OpenStack in a snap ()

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -14,200 +14,221 @@
...
       </div>
-    </div>
   </section>
-
   <section class="p-strip">
     <div class="row">
       <div class="p-notification--caution">
         <p class="p-notification__response">
-          <span class="p-notification__status">Note:</span> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>.
+          <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For information about commercial support for MicroStack, contact <a class="p-link--external" href="https://canonical.com">Canonical</a>.
```
Comment (bethcollins92) [inline templates/index.html:34]: On the copy doc the canonical link points to contact us

Comment (minkyngkm) [reply templates/index.html]: Thanks for reviewing! Fixed. :)

File: templates/index.html [hunk 2]
```diff
@@ -217,50 +238,42 @@ <h3 class="p-stepped-list__title">Launch a VM</h3>
           </li>
         </ol>
         <p>
-          That’s it! Your OpenStack is ready.
+          That’s it! Your micro cloud is ready.
         </p>
       </div>
-      <div class="col-6 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
+      <div class="col-8 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
```
Comment (bethcollins92) [inline templates/index.html:244]: The code is still scrolling here - could you change it to a col-9?
```suggestion
      <div class="col-9 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
```

File: templates/index.html [hunk 3]
```diff
@@ -270,49 +283,42 @@ <h3 class="p-stepped-list__title">Launch a VM</h3>
           </li>
         </ol>
         <p>
-          That’s it! Your OpenStack is ready.
+          That’s it! Your micro cloud is ready.
         </p>
       </div>
-      <div class="col-6 js-tab__content" id="tab-three__content" role="tabpanel" aria-labelledby="tab-three">
+      <div class="col-8 js-tab__content" id="tab-three__content" role="tabpanel" aria-labelledby="tab-three">
```
Comment (bethcollins92) [inline templates/index.html:289]:
```suggestion
      <div class="col-9 js-tab__content" id="tab-three__content" role="tabpanel" aria-labelledby="tab-three">
```

File: templates/index.html [hunk 4]
```diff
@@ -322,13 +328,40 @@ <h3 class="p-stepped-list__title">Launch a VM</h3>
...
+          That’s it! Your micro cloud is ready.
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light">
+    <div class="row u-vertically-center">
+      <div class="col-6">
+        <h2>The team behind MicroStack</h2>
+        <p>MicroStack is maintained by the <a class="p-link--external" href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a class="p-link--external" href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres.</p>
```
Comment (bethcollins92) [inline templates/index.html:340]:
```suggestion
        <p>MicroStack is maintained by the <a class="p-link--external" href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a class="p-link--external" href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres. MicroStack was built by the dedicated OpenStack team at Canonical for the developer community. We also make the Charmed OpenStack for multi-cloud scalable clusters. Please contact us if you have any questions.</p>
```

File: static/js/typer.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
+// Copied from this project:
+// https://steven.codes/typerjs/docs/index.html
+
+var Typer = function (element) {
```
Comment (bethcollins92) [inline static/js/typer.js:4]: Would you be able to make it so the `sudo` doesn't delete as it's the same in both commands? we have an example in this repo doing the same thing https://github.com/canonical-web-and-design/jaas.ai/blob/master/static/js/cli-typer.js (This has more options in the array but you could pretty much copy it). Here it is on [live](https://jaas.ai/)

File: templates/index.html [hunk 6]
```diff
@@ -14,200 +14,221 @@
...
         </p>
       </div>
     </div>
     <div class="row">
-      <div class="col-8">
-        <h2>OpenStack in a snap</h2>
-        <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+      <div class="col-8" style="margin-top: 2rem;">
+        <h2>Straightforward installation</h2>
+        <h4>Fully functional OpenStack in 2 commands and ~20 minutes.</h4>
```
Comment (bethcollins92) [inline templates/index.html:41]: I know you didn't change this but we shouldn't jump from a h2 to a h4, you can add the classname "p-heading--4" to a h3 so it remains semantically correct with also the correct style.
```suggestion
        <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h4>
```

File: templates/index.html [hunk 7]
```diff
@@ -14,200 +14,221 @@
...
+      <div class="col-4 p-divider__block u-vertically-center">
+        <h3 class="u-align--center" style="margin-left: 1.5rem; margin-right: 1.5rem;">~1,500 active installations</h3>
       </div>
     </div>
+  </section>
+  <section class="p-strip">
+      <div class="row">
+        <div class="col-6">
+          <h2>OpenStack for the edge</h2>
+          <h4>Comprehensive cloud platform for distributed low-latency applications</h4>
```
Comment (bethcollins92) [inline templates/index.html:66]: Same here with the heading hierarchy, shouldn't be h2 then h4 (could you check the page for any of the same and replace the h4 with `<h3 class="p-heading--4">`)

File: templates/index.html [hunk 8]
```diff
@@ -14,200 +14,221 @@
...
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-7">
+        <h2>OpenStack on rails</h2>
+        <h4>OpenStack has tens of components and options. Not everyone needs this complexity.</h4>
+       </div>
+      <div class="col-6">
+        <p>MicroStack includes core OpenStack services and the most popular compute, network and storage options. This eliminates the OpenStack complexity and provides an “on rails” experience.</p>
+      </div>
+      <div class="col-8 u-equal-height">
```
Comment (bethcollins92) [inline templates/index.html:162]: There are too many cols in this row - you can only have a maximum of 12 cols in every row. Can you close the row before the col-8? Then I don't think you need the col-8 you can just replace this with a row and then each of the list cols can be a col-3? Try that and let me know if you need help

**Review by bethcollins92 — CHANGES_REQUESTED** [review]

**Review by bethcollins92 — COMMENTED** [review]
Great work Min! 🎉 There are some comments in the code - let me know if anything doesn't make sense

File: templates/index.html (hunk 4 above)
Comment (minkyngkm) [reply templates/index.html:340]: Ohhh followed the texts on design here. Fixed!

File: templates/index.html (hunk 6 above)
Comment (minkyngkm) [reply templates/index.html:41]: Didn't know that we shouldn't jump from a h2 to a h4. Thanks! Fixed. 👍

File: static/js/typer.js (hunk 5 above)
Comment (minkyngkm) [reply static/js/typer.js:4]: Would it be okay just to take the `$ sudo` out of the `span`? The`$ sudo` stays now!

File: templates/index.html (hunk 8 above)
Comment (minkyngkm) [reply templates/index.html]: Fixed! :)

**Review by bethcollins92 — APPROVED** [review]
LGTM Min - good job! 🥇

**Pull request merged**